apiary.apib

```diff
@@ -4307,3 +4307,7 @@ Unregister a webhook.
                 + `bankers_draft`
                 + `bank_transfer`
                 + `payment_card`
+
+## Duration (object)
++ unit: `s` (string, required) - Seconds
```

Comment:
Q: Why not do it like: 
```
+ unit: `s` (enum[string], required)
    + Members
        + s - Seconds
```

We _"did"_ it for Distance on Work Orders.
https://github.com/teamleadercrm/api-internal/pull/60/files#diff-1841f13b750b4c8d06587142329fe64cR177-R181

Reply:
`DurationUnit` also provides minutes, as `min` so that would actually make it:
```
+ unit: `s` (enum[string], required)
    + Members
        + s - Seconds
		+ min - Minutes
```

Reply:
We already use duration in minutes on some endpoints, so having this support it would be ideal IMO.

Reply:
And not in the scope of this proposal, but the `/tasks.create` already uses a duration like structure on `estimated_duration`. 
That could be a good example of it's usage already.

Reply:
Agreeing with @joaoinacio and @tiagobastos. Let's define `unit` as an enum so we are not fixed to having just one possible unit.